src/ios-deploy/ios-deploy.m

```diff
@@ -1819,11 +1887,12 @@ int main(int argc, char *argv[]) {
         { "list_bundle_id", no_argument, NULL, 'B'},
         { "no-wifi", no_argument, NULL, 'W'},
         { "detect_deadlocks", required_argument, NULL, 1000 },
+        { "json", no_argument, NULL, 'j'},
         { NULL, 0, NULL, 0 },
     };
     int ch;
 
```

Comment:
I would append the string with 'j' instead of appending '::j'. The effect here is you are specifying`W::`, that `-W` takes an optional parameter. I know that this opt string stuff is not exactly intuitive... (you could also prepend the string with 'j', same effect)